### ISSUE TITLE
2110 SDP structure violates RFC

### ISSUE THREAD

**Issue by tkernen** [SMPTE/rp2110-23#1]
Details about the SDP structure in rp 2110-23 are available here
https://github.com/AMWA-TV/sdpoker/pull/12

**Comment by jmailhot** [comment]
I agree with the observation that 2110-23 should be amended to be correct (or to say less and let the other standards speak for themselves) as mr @tkernen points out in his related comments.  At some point 2110-23 will come up for 5-year review and the TC can decide whether to improve it or withdraw it.  Are you aware of any current use of this RP in the wild?  I believe the prevalence of 25GBE interfaces has largely eliminated the practice documented in RP2110-23.  I recommend we add this item to the list of things that should be addressed in future revisions, and then close the issue.
